packages/next/compiled/@next/react-refresh-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@next/react-refresh-utils",
-  "version": "12.0.9-canary.0",
+  "version": "12.0.9-canary.1",
```

Comment:
Could this change be reverted please?

Reply:
Done - no idea what this commit is about, I only clicked on the "Update branch" on Github!